Lexdao-Constitutionv02.md

```diff
@@ -0,0 +1,160 @@
+
+If you have received any LDC or are otherwise a DAO Member, you consent to contact and agree to become legally bound by this Agreement as both a LexDAO participant and more specifically a "LexDAO Member".
+
+**Section 2.5 - Limitations.**  Status as a LexDAO Member does not (and shall not be deemed to) create, and the LexDAO Membership Rights do not (and shall not be deemed to) include, any authority, right or power on the part of a LexDAO Member to act as the agent, representative or attorney of or otherwise act on behalf of LexDAO or any other LDC Member, to bind LexDAO or any other LDC Member to any Contract or Liability or to convey any LexDAO property or any asset, right or property owned or held by or on behalf of LexDAO or any LDC Member. Without limiting the generality of the foregoing, no LDC Member shall be deemed the partner of LexDAO or any other LDC Member. No LDC Member shall state, purport, imply, hold out or represent to any person that such LDC Member or any other LDC Member has any such authority, right or power.
+
+To the maximum extent permitted by applicable law, no LDC Member shall be (or shall be deemed to be) liable for any liability of the LexDAO or any other LDC Member. This shall not (and shall not be deemed to) create or imply any obligation of LexDAO or any LDC Member to indemnify or compensate any LDC Member from, or hold any LDC Member harmless against, any Liabilities incurred by such DAO Member under any applicable law, in connection with the LDC Member's participation in LexDAO or otherwise.
+
+**Section 2.6 - Inactive status.** A member who falls from good standing may have their membership revoked or suspended through the vote of LDC Membership (each said action, a “Guild Kick/Member Removal”).  References herein to the rights and entitlements of members shall be understood to refer only to members in good standing.
+
+**Section 2.7 - Access to information.** Members shall have access to information concerning the operational and financial affairs of via the Treasury application of the organization.  Currently the LexDAO treasury can be viewed via Designated Smart Contract.  
+
+**Section 2.8 - Settlement of disputes.** In any dispute between LexDAO and any of its members or former members which cannot be resolved through informal negotiation, it shall be the policy of LexDAO to prefer the use of mediation whereby an impartial mediator may facilitate negotiations between the parties and assist them in developing a mutually acceptable settlement. Neither party with a grievance against the other shall have recourse to litigation until the matter is submitted to mediation and attempted to be resolved in good faith.  All LDC Membership agree that there is a preference to settle disputes amongst members or between members and LexDAO via decentralized dispute mechanisms via smart contract protocols. 
```

Comment:
Is there a Sergeant at Arms or similar position who is directly responsible for the management of disputes?  Didn't see it in the  Ops Operating System